[BUG] Build fails on latest nightly

**Describe the bug**
Build fails on latest nightly:

```
error[E0557]: feature has been removed
   --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/elasticsearch-7.12.0-alpha.1/src/lib.rs:352:39
    |
352 | #![cfg_attr(RUSTC_IS_NIGHTLY, feature(external_doc))]
    |                                       ^^^^^^^^^^^^ feature has been removed
    |
    = note: use #[doc = include_str!("filename")] instead, which handles macro invocations
```

**To Reproduce**
Steps to reproduce the behavior:
1. Use rustc 1.54.0-nightly (35fff69d0 2021-06-06)
2. Try to build.

**Expected behavior**
Success.

**Environment (please complete the following information):**
 - OS: Ubuntu
 - rustc version: x86_64-unknown-linux-gnu 1.54.0-nightly (35fff69d0 2021-06-06)

--- a/elasticsearch/src/lib.rs
+++ b/elasticsearch/src/lib.rs
@@ -347,11 +347,19 @@
 // TODO: turn on before releasing :) Will require adding documentation within all REST API specs
 // #![deny(missing_docs)]
 
-// also test examples in README when using rust nightly.
-// required as external_doc feature requires nightly
-#![cfg_attr(RUSTC_IS_NIGHTLY, feature(external_doc))]
-#[cfg_attr(RUSTC_IS_NIGHTLY, doc(include = "../../README.md"), cfg(doctest))]
-type _DoctestReadme = ();
+// also test examples in README
+// source: https://github.com/rust-lang/cargo/issues/383#issuecomment-720873790
+#[cfg(doctest)]
+mod readme {
+    macro_rules! external_doc_test {
+        ($x:expr) => {
+            #[doc = $x]
+            extern "C" {}
+        };
+    }
+
+    external_doc_test!(include_str!("../../README.md"));
+}
 
 #[macro_use]
 extern crate dyn_clone;

